app/controllers/posts_controller.rb

```diff
@@ -7,5 +7,30 @@ def index
   def show
     @user = User.find(params[:user_id])
     @post = @user.posts.find(params[:id])
+    @like = Like.new
+  end
+
+  def new
+    @user = User.find(params[:user_id])
+    @post = @user.posts.new
+  end
+
+  def create
+    @user = User.find(params[:user_id])
+    @post = @user.posts.new(post_params)
+    @post.author = current_user
+    @post.commentsCounter = 0
+    @post.likesCounter = 0
+    if @post.save
+      flash[:success] = 'Post created!'
+      redirect_to user_posts_path(current_user)
+    else
+      flash[:error] = 'Post not created!'
+      render :new
+    end
+  end
+
+  def post_params
+    params.require(:post).permit(:title, :content)
   end
 end
```

Comment:
- [OPTIONAL] I would recommend you to make your post_params `private`. You can chek this [link](https://tjoye20.medium.com/ruby-access-control-basics-public-vs-private-vs-protected-methods-7788b26e04a7#:~:text=Private%20methods%20can%20only%20be,receiver%20is%20always%20implicitly%20self.) for more info.